### PR TITLE
[PHP] add one framework

### PR DESCRIPTION
**Pull request by lizhichao**
Before submitting your PR, please review the following checklist :

## If you are adding a framework

+ [x] A `Dockerfile` exists, or can be built ?
+ [x] I've updated `CI` config file (and **commited** `.travis.yml`)
~~~sh
bin/make ci_config
~~~
+ [x] I've updated `.dependabot` config file  (and **commited** `.dependabot/config.yml`)
~~~sh
bin/make deps_config
~~~
+ [x] All tests passes ?
~~~
export FRAMEWORK=<MY_FRAMEWORK>
shards install
shards build --release
bin/make config
bin/neph ${FRAMEWORK}
crystal spec
~~~

### CODE REVIEW

File: php/one/config.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+framework:
+  website: https://github.com/lizhichao/one
+  version: 1.8.2
```
Comment (waghanza) [inline php/one/config.yaml]: please use X.Y format (`1.8`)

File: php/one/config.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+framework:
+  website: https://github.com/lizhichao/one
```
Comment (waghanza) [inline php/one/config.yaml]: use the short cut `github: lizhichao/one`

File: php/one/composer.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  ],
+  "license": "Apache-2.0",
+  "authors": [
+    {
+      "name": "tanszhe",
+      "email": "1018595261@qq.com"
+    }
+  ],
+  "require": {
+    "php": ">=7.1.0",
```
Comment (waghanza) [inline php/one/composer.json]: not required

File: php/one/composer.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    "lizhichao/one": ">=1.8.2"
```
Comment (waghanza) [inline php/one/composer.json]: please us `~1.8.2` instead, the purpose is to ensure that we stick to `1.8`

**Comment by lizhichao** [comment]
Thank you for check . @waghanza 

I made a change. Is that OK?

**Comment by waghanza** [comment]
@lizhichao I've made a `PR` on your fork

**Comment by lizhichao** [comment]
@waghanza MySQL config has been deleted

**Comment by lizhichao** [comment]
This branch has conflicts that must be resolved

**Pull request closed without merging**